### PR TITLE
list_of_events: Add metacache events

### PR DESCRIPTION
**Pull request by sirainen**
DOV-3586 DOV-3591 DOV-3590 DOV-3588 DOV-3587 DOV-3589

### CODE REVIEW

**Comment by cmouse** [comment]
@sirainen this needs manual rebase.

**Comment by sirainen** [comment]
rebased

File: source/conf.py [hunk 1]
```diff
@@ -46,7 +46,6 @@
     'sphinx.ext.mathjax',
     'sphinx.ext.ifconfig',
     'sphinx.ext.todo',
-    'sphinx_copybutton',
```
Comment (cmouse) [inline source/conf.py]: why are you removing copybutton?

Comment (sirainen) [reply source/conf.py]: Accident. I hadn't bothered installing that so I always kept deleting it. But a couple of days ago I installed it so shouldn't have such problems anymore.

File: source/admin_manual/list_of_events.rst [hunk 2]
```diff
@@ -1574,6 +1597,179 @@ renumbering was done after all.
...
+
+User or mailbox index bundle file is uploaded. These events can happen while
+the user or mailbox is being uploaded. Note that the metacache_user_* events
+can also be inherited from a mailbox event and include the mailbox fields if
+the user upload was triggered by a mailbox upload.
+
++-----------------------+------------------------------------------------------+
+| Field                 | Description                                          |
++=======================+======================================================+
+| fname                 | Bundle filename                                      |
```
Comment (cmouse) [inline source/admin_manual/list_of_events.rst]: i know the code is probably already committed but is there a reason we can't use "filename" here instead?

File: source/admin_manual/list_of_events.rst [hunk 3]
```diff
@@ -1574,6 +1597,179 @@ renumbering was done after all.
...
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+metacache_user_bundle_upload_finished
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+metacache_mailbox_bundle_upload_started
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+metacache_mailbox_bundle_upload_finished
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+User or mailbox index bundle file is uploaded. These events can happen while
+the user or mailbox is being uploaded. Note that the metacache_user_* events
```
Comment (cmouse) [inline source/admin_manual/list_of_events.rst]: ``metacache_user_* `` needs double-backtick fencing

File: source/admin_manual/list_of_events.rst [hunk 4]
```diff
@@ -1574,6 +1597,179 @@ renumbering was done after all.
...
+metacache_mailbox_bundle_download_finished
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+User or mailbox index bundle file is downloaded. These events can happen while
+the user or mailbox is being refreshed.
+
++-----------------------+------------------------------------------------------+
+| Field                 | Description                                          |
++=======================+======================================================+
+| fname                 | Bundle filename                                      |
```
Comment (cmouse) [inline source/admin_manual/list_of_events.rst]: same filename comment

**Pull request merged**